examples/maintained-notebooks.txt

```diff
@@ -11,8 +11,8 @@ entanglement_swapping
 expectation_value_example
 Forest_portability_example
 measurement_reduction_example
-qiskit_integration
+# qiskit_integration FIXME https://github.com/CQCL/pytket/issues/103
```

Comment:
The comment symbol is not parsed by the `check-examples` script so this will give an error.

I think a better solution is to edit the `check-examples` script so that it skips _running_ these two tests (it can still do the check on the validity of the notebook).

Reply:
And add a reference to the two bugs there.